modules/container-base/README.md

```diff
@@ -0,0 +1,56 @@
+# Dataverse Base Container Image
+
+A "base image" offers you a pre-installed and pre-tuned application server to deploy Dataverse software to.
+Adding basic functionality like executing scripts at container boot, monitoring, memory tweaks, etc., is all done
+at this layer, to make the application image focus on the app itself.
+
+## Quick Reference
+
+**Maintained by:** 
+
+This image is created, maintained and supported by the Dataverse community on a best-effort basis.
+
+**Where to find documentation:**
+
+The [Dataverse Container Guide - Base Image](https://guides.dataverse.org/en/latest/container/base-image.html)
+provides in-depth information about content, building, tuning and so on for this image. 
+
+**Where to get help and ask questions:**
+
+IQSS will not offer you support how to deploy or run it. Please reach out to the community for help on using it.
+You can join the Community Chat on Matrix at https://chat.dataverse.org or the Community Slack at
+https://dataversecommunity.slack.com to ask for help and guidance.
+
+## Supported Image Tags
+
+This image is sourced within the main upstream code [repository of the Dataverse software](https://github.com/IQSS/dataverse).
+Development and maintenance happens there (again, by the community). Community-supported image tags are based on the two
+most important branches:
+
+- `develop` representing the unstable state of affairs in Dataverse's development branch
+  ([`Dockerfile`](https://github.com/IQSS/dataverse/tree/develop/modules/container-base/src/main/docker/Dockerfile))
+- `release` representing the latest stable release in Dataverse's main branch
```

Comment:
You've lost me. Shouldn't this be `master` rather than `release`?

Reply:
Oh, you're talking about tags here. Hmm. I find it confusing that one of the tags matches a branch ("develop") and another tag does not ("release" vs "master"). I would find it less confusing if neither tag matched. For tags, we could try "stable" and "unstable" for example.

Something like this maybe:

---

This image is sourced from the main upstream code [repository of the Dataverse software](https://github.com/IQSS/dataverse). Development and maintenance of the [image code](https://github.com/IQSS/dataverse/tree/develop/modules/container-base/src/main/docker/Dockerfile) happens there (again, by the community). Community-supported image tags are based on the two most important upstream branches:

- The `unstable` tag corresponds to the `develop` branch, where pull requests are merged.
-  The`stable` tag corresponds to the `master` branch, where releases are cut from.

Reply:
Alright, will do!

Reply:
Applied as requested.

Reply:
From a quick glance, looks good! Thanks!